airlock/templates/file_browser/index.html

```diff
@@ -108,6 +108,7 @@
       {% elif current_request %}
         {% #button variant="success" type="link" href=current_request.get_url id="current-request-button" %}Current release request{% /button %}
       {% endif %}
+      {% #button type="link" href=workspace.get_requests_url variant="success" id="requests-workspace-button" %}All release requests for Workspace{% /button %}
```

Comment:
Are we consistent with capitalising "Workspace"?  (In general I don't think we should capitalise Nouns that happen to be the Names of Objects in our Domain...)